third_party/2and3/flask/app.pyi

```diff
@@ -26,25 +27,25 @@ class Flask(_PackageBoundObject):
     app_ctx_globals_class: type = ...
     config_class: Type[Config] = ...
     testing: Any = ...
-    secret_key: Any = ...
+    secret_key: Union[Text, bytes, None] = ...
     session_cookie_name: Any = ...
-    permanent_session_lifetime: Any = ...
-    send_file_max_age_default: Any = ...
+    permanent_session_lifetime: timedelta = ...
+    send_file_max_age_default: timedelta = ...
     use_x_sendfile: Any = ...
     json_encoder: Any = ...
     json_decoder: Any = ...
-    jinja_options: Any = ...
+    jinja_options: Dict[str, Any] = ...
     default_config: Any = ...
     url_rule_class: type = ...
     test_client_class: type = ...
     test_cli_runner_class: type = ...
     session_interface: Any = ...
     import_name: str = ...
     template_folder: str = ...
-    root_path: Any = ...
+    root_path: str = ...
```

Comment:
As this is initialized to `None`, this should be `Optional`. Also I guess this can be `unicode` in Python 2, so using `Optional[Union[str, Text]]` seems better to me.